### ISSUE TITLE
Adding moko-resources as dependency breaks compose-multiplatform-ios-android-template

### ISSUE THREAD

**Issue by nicolashaan** [icerockdev/moko-resources#590]
### Steps to reproduce
1. checkout https://github.com/JetBrains/compose-multiplatform-ios-android-template
2. Load the _iOS_ project template into _Xcode_. Check that the sample is compiling and running. Press _"Hello World!"_ displays an image
3. Modify `shared/build.gradle.kts`:
```
plugins {
    kotlin("multiplatform")
    id("com.android.library")
    id("org.jetbrains.compose")
    **id("dev.icerock.mobile.multiplatform-resources") version "0.23.0"**
}

//....

        val commonMain by getting {
            dependencies {
                implementation(compose.runtime)
                implementation(compose.foundation)
                implementation(compose.material)
                @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
                implementation(compose.components.resources)
                **implementation("dev.icerock.moko:resources:0.23.0")
                implementation("dev.icerock.moko:resources-compose:0.23.0")**
            }
        }
```

4. do _Product/Clean build folder_ in Xcode
5. Build and run the iOS App

**Expected**
The image is displayed as in step 2

**Bug**
The app is crashing with:
```
Uncaught Kotlin exception: org.jetbrains.compose.resources.MissingResourceException: Missing resource with path: compose-multiplatform.xml
```

**Comment by Ustimov** [comment]
Kotlin 1.9 and Compose 1.5 aren't supported yet

**Comment by DonBin** [comment]
I am wondering when Kotlin 1.9 and Compose 1.5 will be supported, thx

**Comment by estivensh4** [comment]
how sad that it has already been 4 months and they are still trying to upgrade a version of kotlin to 1.9, which is affecting many of us.

**Comment by KentVu** [comment]
I'm still hang on to this https://github.com/icerockdev/moko-resources/pull/575 to be able to upgrade to Kotlin 1.9.

**Comment by micHar** [comment]
@Ustimov can you elaborate what breaks in kotlin 1.9? Maybe we could help?

I was trying to use moko-resource but unfortunately got 
Exception in thread "main" java.util.MissingResourceException: Can't find bundle for base name localization/plcomexample_mokoBundle, locale pl_PL.

Not sure if it's the same reason, though.

**Comment by estivensh4** [comment]
> I'm still hang on to this #575 to be able to upgrade to Kotlin 1.9.

mas facil creare mi propia libreria ya que estos cambios son una eternidad aparte veo muy deficiente la documentacion a pesar de la gran comunidad que tienen y lo importante que es esta libreria para Kotlin Multiplatform.

**Comment by estivensh4** [comment]
> I'm still hang on to this #575 to be able to upgrade to Kotlin 1.9.

I will create my own library easier because these changes are an eternity apart I see very poor documentation despite the great community they have and how important is this library for Kotlin Multiplatform.

**Comment by kurthuwig** [comment]
I've got it working:

moko-resources disables the copy of the resources into the iOS directory. I've added this "Run script" in the build configuration and now it works again:

    mkdir -p "$BUILT_PRODUCTS_DIR"/"$CONTENTS_FOLDER_PATH"/compose-resources
    cp "$SRCROOT/../composeApp/src/commonMain/resources"/*.xml \
        "$BUILT_PRODUCTS_DIR"/"$CONTENTS_FOLDER_PATH"/compose-resources

**Comment by Alex009** [comment]
#575 should fix compatibility with compose multiplatform resources engine

**Comment by ExNDY** [comment]
Sample is archivated, in 0.24.0-alpha-6 compose multiplatform supported

**Comment by Alex009** [comment]
should be fixed in 0.24.0-beta-1
